effekt/jvm/src/test/scala/effekt/EffektTests.scala

```diff
@@ -59,6 +61,27 @@ trait EffektTests extends munit.FunSuite {
     compiler.compileFile(input.getPath, configs)
     compiler.messaging.get.toList
 
+  /**
+   * Normalizes test output to ignore certain errors.
+   * 
+   * Currently:
+   * - If on Windows, replaces Windows paths with unix-style paths
+   *   `C:\\Documents` -> `c/Documents`, `examples\neg\Coverage.effekt` -> `examples/neg/Coverage.effekt`
+   */
+  def normalizeOutput(s: String): String = os match {
+    case OS.POSIX => s
+    case OS.Windows =>
+      // Regex for windows path-like substrings, captures: \1 drive letter (optional), \2 directories
+      val windowsPathRegex = """(?:([a-zA-Z]):\\{1,2})?((?:[^\\/:*?"<>|\r\n]+\\)*[^\\/:*?"<>|\r\n]*)""".r
+      windowsPathRegex.replaceAllIn(s, m => {
+        val drive = Option(m.group(1)) // Retrieve the drive letter or an empty string if not present
+        val path = m.group(2).replaceAll("\\\\", "/") // Replace backslashes with forward slashes in directories
+        drive match {
+          case Some(d) => s"$d/$path" // Combine the drive letter (if present) and the Unixified path
+          case None => path
+        }
+      })
+  }
```

Comment:
While `munit` does some normalization/ignores some things in `assertNoDiff`, I could not find a way to hook into this.
So now, depending on current OS, I normalize things that look like paths.

Reply:
Doesn't look this this could be customized in munit: 

https://github.com/scalameta/munit/blob/cd435cb2cb0f8784f5de65c64355040e852cef27/munit/shared/src/main/scala/munit/internal/difflib/Diffs.scala#L34-L58